Gemfile

```diff
@@ -46,14 +46,19 @@ gem 'devise', '~> 4.5.0'
 
 # Login authenticacion
 gem 'devise_token_auth', '~> 1.0.0.rc2'
+
+# Json serializer
+gem 'active_model_serializers', '~> 0.10.0'
 #####
 
+gem 'wor-paginate', '~> 0.1.4'
+
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'rspec-rails', '~> 3.8'
   # Para crear factories en los modelos
-  gem 'factory_bot_rails'
+  gem 'factory_bot_rails', :require => false
```

Comment:
I do not undersand why did you add this?

Reply:
I had a duplicate factory at one point and didn't realize so I had to add it for FactoryBot to work. I'll delete it now.